src/probnum/randvars/_constant.py

```diff
@@ -1,4 +1,5 @@
 """(Almost surely) constant random variables."""
```

Comment:
I did not notice this before, but I'm not sure if we want exactly this comment here. This reads more like "I'm pretty sure that this module contains constant random variables, but we can never know"

Reply:
Changed it to a better description.